check_stability.py

```diff
@@ -26,6 +27,10 @@
 wptrunner = None
 
 
+reload(sys)
+sys.setdefaultencoding('utf8')
```

Comment:
This is a hack that seems to be considered bad form (and isn't python 3 compatible). Let's explicitly encode things instead, as required.